src/client/eppo-client.ts

```diff
@@ -495,11 +482,23 @@ export default class EppoClient {
       // Check if the assigned variation is an active bandit
       // Note: the reason for non-bandit assignments include the subject being bucketed into a non-bandit variation or
       // a rollout having been done.
+      const banditVariations = this.banditVariationConfigurationStore?.get(flagKey);
       const banditKey = banditVariations?.find(
         (banditVariation) => banditVariation.variationValue === variation,
       )?.key;
 
       if (banditKey) {
+        // If no actions are passed, return the default value and do not log a bandit assignment
+        if (!Object.keys(actions).length) {
+          return {
+            ...defaultResult,
+            evaluationDetails: flagEvaluationDetailsBuilder.buildForNoneResult(
+              'NO_ACTIONS_SUPPLIED_FOR_BANDIT',
+              'No actions passed for a flag that evaluated to a bandit variation',
```

Comment:
Per the [Slack thread](https://eppo-group.slack.com/archives/C05R6BK5BB9/p1721149341928549), we'll now want to return the bandit variation with a null action. We can keep the same reason, though. This will probably require updating of shared sdk test data.